Basic deployable version (#1)
NBISweden/cytomine-bp-helm: README.md

instructions for using the
[ingress dns](https://minikube.sigs.k8s.io/docs/handbook/addons/ingress-dns)
add-on.

### Setting up the environment on OS X

Minikube doesn't seem to be superhappy on OS X, but with a few precautions it
works!

-  If you wish to use the `ingress-dns` addon (which is used in this project),
   you need to use the `hyperkit` driver (start minikube with:
   `minikube start --driver=hyperkit`). I tried using the docker driver but
   couldn't get it to work on my machine.

-  Connecting the cisco VPN client seems to mess with minikubes networking. I
   need to restart minikube every time I need to use the VPN.

-  Docker desktop also seems to conflict with minikube, so I can't run both
   docker and minikube at the same time (unless I use the docker driver, but
   then ingress-dns doesn't work).

### Setting up the environment on Ubuntu System

We need to add the `minikube-ip` as a DNS server

Update the file `/etc/resolvconf/resolv.conf.d/base` to have the following contents

```
search test
nameserver <minikube ip>
timeout 5
```

Ensure your Linux OS uses `systemcl`, run the following command

`sudo resolvconf -u`

> To verify, run the following command to check if the `/etc/resolv.conf` is updated with the minikube ip.

Reviewer: The verifying command was missing (I assume `cat /etc/resolv.conf` ?). 
```suggestion
To verify, check that the `/etc/resolv.conf` is updated with the minikube ip.
```

> 

Reviewer: Strangely, using `resolvconf` in Ubuntu for adding the DNS did not work for me, but the following quick fix did:
````suggestion

If the above steps do not work or you do not want to use `resolvconf`, add the above contents manually instead. *After** `minikube` is started run
```
echo -e $"\nsearch test\nnameserver $(minikube ip)\ntimeout 5" \
| sudo tee -a /etc/resolv.conf
```
When you are done testing cytomine, you can cleanup `resolv.conf` or wait till this is done automatically when the dhcp lease is renewed.

````
